src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -1816,10 +1805,7 @@ void SolverInterfaceImpl::clearMappings(std::vector<MappingContext> &contexts)
   // Clear non-stationary, non-incremental mappings
   using namespace mapping;
   for (impl::MappingContext &context : contexts) {
-    bool isStationary = context.timing == MappingConfiguration::INITIAL;
-    if (not isStationary) {
-      context.mapping->clear();
-    }
+    context.mapping->clear();
```

Comment:
This seems wrong. Doesn't this lead to a re-computation of the mapping in every iteration?

Reply:
This would clear mapping if we call `SolverInterfaceImpl::clearMappings`, which seems to be the way to go here.

Reply:
By dropping the timings, preCICE can internally handle the mappings whenever it needs to (Partitioning), meaning there is also no need to collectively clear mappings. Hence, the complete `clearMappings` function can go.

Reply:
Done